docs/how-to-guides/environment-setup-and-maintenance/adding-usgscsm-plugin-to-isis.md

```diff
@@ -0,0 +1,78 @@
+# Adding USGSCSM Plugin to ISIS
+
+## For users
+You can install the latest [USGSCSM](https://github.com/DOI-USGS/usgscsm) library via conda:
+
+```sh
+conda install conda-forge::usgscsm
+```
+## For developers
+
+### Plugin setup
+Update `IsisPreferences` file under `Group = Plugins` and add `"$CONDA_PREFIX/lib/csmplugins/"`, for example:
+
+  ```
```

Comment:
We might want to include in what contexts adding the directory here is necessary for.  E.g. if testing a new plugin or using older versions of USGSCSM/ISIS where the defaults dont agree.